### PR TITLE
add type filter to GET /changes

### PR DESCRIPTION
**Pull request by adropofilm**
**Description**
- Implemented type filter for GET /changes so changes made after X date/datetime can be filtered by `type` (`sub_type` under the hood)

**Questions**
1. I need to update the docs for this endpoint. I noticed we're using stoplight, any tips? :)

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:49184

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:49185

File: app/models/resource_content.rb [hunk 1]
```diff
@@ -101,4 +101,11 @@ def self.changes(updated_after)
     end
   end
 
+  def self.filter_subtype(type = nil)
+    if type.present?
+      where sub_type: type
+    else
+      all
```
Comment (naveed-ahmad) [inline app/models/resource_content.rb:108]: self

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:49186

**Review by naveed-ahmad — APPROVED** [review]

**Pull request merged**